release-notes.txt

```diff
@@ -1,3 +1,688 @@
+    * sys/gnrc_sixlowpan_frag_rb: separate out classic frag specific code
+    * sys/gnrc_sixlowpan_frag_rb: fix memory-leak in _rm_by_datagram() and in interval marker inherited from base
+    * sys/gnrc_sixlowpan_frag: various optimizations on sending
+    * sys/gnrc_sixlowpan_iphc: add fragment forwarding stubs 
+    * sys/gnrc_sixlowpan_iphc_nhc: determine UDP hdr length from RB, fix fragmented data size allocation
+    * sys/gnrc_tcp: return immediately on gnrc_tcp_recv if a connection is closing 
+    + sys/net/gcoap: add macro to delay initialization of gcoap
+    * sys/net/gcoap: do not allocate RX buf on stack
+    * sys/pthread: check return value of malloc(), prevent NULL pointer deference
+    * sys/shell/gnrc_netif: Use netif API for ifconfig
+    * sys/shell/sc_nimble`_netif: allow connecting by name
+    + sys/sock: initial definitions for asynchronous event handling
+    + sys/sock_async: initial import of event-based implementation
+
+Packages
+--------
+    * pkg: cleanup management of dependencies
+    * pkg/pkg.mk: use intermediate state files 
+    * pkg/ccn-lite: bump version
+    + pkg/flatbuffers: add support for FlatBuffers serialization library
+    * pkg/gecko_sdk: update to version 2.7
+    * pkg/libfixmath: several improvements and fixes to support 8bit
+    * pkg/littlefs: bump littlefs version to 1.7.2
+    * pkg/lora-serialization: bump to latest version and update test for AVR
+    * pkg/lwip: add support for esp32 Ethernet device
+    * pkg/lwip: add IPv4 support for LWIP-stm32
+    * pkg/lwip: start DHCP for a netif with lwip_dhcp
+    * pkg/lwip: add STM32 support
+    * pkg/lwip: enable lwIP for ESP8266
+    + pkg/nanopb: add nanoPb protocol buffers library package
+    * pkg/nimble: bump version
+    + pkg/nimble: add simple BLE connection manager: autoconn
+    + pkg/nimble/autoconn: add user event callback
+    + pkg/nimble_netif: catch L2CAP connection failures
+    + pkg/nimble_netif: add additional events
+    * pkg/tinydtls: add DTLS sock API implementation
+    * pkg/tinydtls: bump version
+    + pkg/tensorflow-lite: add support to RIOT
+    + pkg/utensor: add support to RIOT
+    * pkg/wakaama: Add basic LWM2M client implementation
+
+Boards
+------
+    - boards: remove RTT_NUMOF/RTC_NUMOF
+    - boards: unexport PROGRAMMER
+    * boards/arduino-zero: configure ADC channels
+    + boards/atmega328p: initial support for standalone version
+    * boards/atmega328p: support variable xtimer frequencies
+    + boards/atmega1284P: initial support for standalone version
+    + boards/atmega256rfr2: initial support
+    + boards/avr-rss2: initial support
+    + boards/cc1252: add initial support
+    * boards/common: clean up msba2 common files
+    + boards/derfmega*: add initial support
+    + boards/esp32-ttgo-t-beam: add initial support
+    - boards/frdm: remove support for OpenOCD < v0.10.0
+    * boards/hifive1b: configure ADC channels
```

Comment:
There is no support for ADC on hifive1b.

Reply:
Must have been a mix up with `boards/arduino-zero: configure ADC channels`

Reply:
Thanks for the catch.